src/RegistryInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Part of the Joomla Framework Registry Package
+ *
+ * @copyright  Copyright (C) 2023 Open Source Matters, Inc.
+ * @license    GNU General Public License version 2 or later; see LICENSE
+ */
+
+namespace Joomla\Registry;
+
+/**
+ * Registry Interface
+ *
+ * @method exists(string $key)
+ * @method get(string $key, $default = null)
+ * @method set(string $key, $value)
+ * @method remove(string $key)
+ *
+ * @since  2.1.0
+ */
+interface RegistryInterface
```

Comment:
Why do you don't add here the functions like get or set?

Reply:
See my comment ;)